Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/sparkjobview.js

```diff
@@ -320,29 +306,48 @@ function getJobResult() {
     });
 }
 
+function renderApplicationGraph() {
+    getMessageAsync('/applications/application_graph', 'spark', function (s) {
+        var yarnAppWithJobs= JSON.parse(s);
+        spark.selectedYarnApp = yarnAppWithJobs.app;
+        spark.currentSelectedJobs = yarnAppWithJobs.jobs;
+        renderJobGraphOnApplicationLevel(spark.currentSelectedJobs);
+    }, spark.appId);
+}
 
-// function setLivyLog() {
-//     getMessageAsync(localhost + projectId + "/?restType=livy&applicationId=" + appId, function (s) {
-//         $("#livyJobLog").text(s);
-//     });
-// }
-
-function setJobDetail() {
-    var selectedApp = findElement(spark.applicationList, function (d) {
-       return d.id === appId;
-    });
-    if(typeof selectedApp === 'undefined') {
-        return;
-    }
-    setDebugInfo("selectApp " + appId);
-    if(selectedApp.attempts[0].sparkUser === 'hive') {
+function renderStageDetails() {
+    if(spark.attemptId === 0) {
```

Comment:
change it to 'spark.attemptId == 0', and I think that's what I am willing to show. Thx.